Makefile

```diff
@@ -0,0 +1,22 @@
+SHELL=/bin/bash
+
+run1:
+	bin/run.sh test1_stream
+run2:
+	bin/run.sh lassen-keras-dbscan_stream
+push:
```

Comment:
An update to the MakeFile should remove the make push command - while bin/push.sh is being removed.

Reply:
Done